File: app/src/main/java/pl/coddev/applu/p/ConfigureActivity.java
package pl.coddev.applu.p;

import android.app.Activity;
import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.EditText;
import android.widget.ImageButton;
import android.widget.Spinner;
import android.widget.TextView;

import java.util.List;

/**
 * Created by pw on 16/03/15.
 */
public class ConfigureActivity extends Activity implements AdapterView.OnItemSelectedListener{
    private static final String TAG = "ConfigureActivity";
    private int appWidgetId;
    private ImageButton confirmButton;
    Context context;
    private Spinner countrySpinner;
    String selectedCountry = "";
    Integer clickCount;
    List<String> countryCodes;
    EditText codeEdit;
    private TextView countrySpinnerTxt;
    ArrayAdapter<String> adapter;
    private boolean groupCodeExists;
    private boolean sourceCountryExists;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);



    }

    @Override
    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
        selectedCountry = parent.getItemAtPosition(position).toString();
    }

    @Override
    public void onNothingSelected(AdapterView<?> parent) {

    }
}

File: app/src/main/java/pl/coddev/applu/ui/ConfigureActivity.kt
package pl.coddev.applu.ui

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.view.View
import android.widget.*
import android.widget.AdapterView.OnItemSelectedListener

/**
 * Created by pw on 16/03/15.
 */
class ConfigureActivity : Activity(), OnItemSelectedListener {
    private val appWidgetId = 0
    private val confirmButton: ImageButton? = null
    var context: Context? = null
    private val countrySpinner: Spinner? = null
    var selectedCountry = ""
    var clickCount: Int? = null
    var countryCodes: List<String>? = null
    var codeEdit: EditText? = null
    private val countrySpinnerTxt: TextView? = null
    var adapter: ArrayAdapter<String>? = null
    private val groupCodeExists = false
    private val sourceCountryExists = false
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onItemSelected(parent: AdapterView<*>, view: View, position: Int, id: Long) {
        selectedCountry = parent.getItemAtPosition(position).toString()
    }

    override fun onNothingSelected(parent: AdapterView<*>?) {}

    companion object {
        private const val TAG = "ConfigureActivity"
    }
}